java/src/jmri/jmrit/decoderdefn/DecoderFile.properties

```diff
@@ -11,4 +11,6 @@ WrongContent    = The file doesn't contain a decoder definition.
 CopyError1      = File couldn't be copied
 CopyError2      = Error while copying file
 
+DecoderProgressMessage = Recreating decoder index...
+DecoderProgressCancel = Cancel
```

Comment:
I suggest that you use `ButtonCancel` instead of `DecoderProgressCancel` since `ButtonCancel` is already defined here:
https://github.com/JMRI/JMRI/blob/master/java/src/jmri/NamedBeanBundle.properties#L351

So you don't need to define `ButtonCancel` in this file since it's already defined upstream. Only change `Bundle.getMessage("DecoderProgressCancel")` to `Bundle.getMessage("ButtonCancel")`.

Reply:
Will do.